docs/site/src/demos/lists/FolderList.js

```diff
@@ -23,11 +24,11 @@ function FolderList(props, context) {
     <div className={classes.root}>
       <List>
         <ListItem button>
-          <Avatar icon={<span className="material-icons">folder</span>} />
+          <Avatar icon={<FolderIcon />} />
```

Comment:
I have the following warning:
> warning.js:36Warning: Failed prop type: Invalid prop `icon` supplied to `Avatar`, expected a ReactNode.
    in Avatar (created by FolderList)
    in FolderList (created by Demo)
    in div (created by Demo)
    in div (created by Demo)
    in Demo (created by MarkdownDocs)
    in div (created by MarkdownDocs)
    in MarkdownDocs (created by RouterContext)
    in div (created by AppContent)
    in AppContent (created by RouterContext)
    in div (created by AppFrame)
    in AppFrame (created by Connect(AppFrame))
    in Connect(AppFrame) (created by withWidth(Connect(AppFrame)))
    in EventListener (created by withWidth(Connect(AppFrame)))
    in withWidth(Connect(AppFrame)) (created by RouterContext)
    in RouterContext (created by Router)
    in ScrollBehaviorContext (created by Router)
    in Router (created by AppRouter)
    in AppRouter (created by App)
    in MuiThemeProvider (created by App)
    in App (created by Connect(App))
    in Connect(App)
    in Provider
    in AppContainer